docs/app/documentation/component-docs/date-picker/date-picker-docs.component.html

```diff
@@ -81,3 +81,11 @@ <h2>Disabled State</h2>
     <fd-date-picker-disabled-example></fd-date-picker-disabled-example>
 </component-example>
 <code-example [exampleFiles]="datePickerDisabled"></code-example>
+
+<separator></separator>
+<h2>Disabled Reactive Form</h2>
```

Comment:
For consistency with the other components, could you name this `Disabled DatePicker In A Reactive Form` ?

Reply:
Yes!